### ISSUE TITLE
Model difference to original code

### ISSUE THREAD

**Issue by juicebox18** [kurapan/EAST#20]
Thank you for the great implementation. Did you by any chance do some comparison of the model output with the argman code you used as a template? I mean anything other than checking for similar precision&recall performance? 

I tried to convert argman's tf.slim based models to your keras style but the topology is very different (mostly because the ResNet implementation with slim's argscope are defined differently). So now I wanted to check if the models "work" the same as with the argman approach if I retrain them with keras. So did you maybe do any layer-by-layer comparisons or reproducibility checks for your new implementation and the argman implementation to verify your approach?

**Comment by kurapan** [comment]
Sorry for a late reply.
I haven't done any thorough comparison because my motivation was to re-implement EAST in Keras and achieve performance similar to the TF implementation, which already achieves better results than those published in the original paper (probably thanks to ResNet50 and dice loss).
I think that the crucial parts (U-net skip connections, loss functions, etc.) are implemented in a similar way, but I have opted for some small changes (e.g. different optimizer). I don't exactly remember all of those small changes anymore, though. Therefore, I expect that the outputs and internal representations of the network might differ a bit, and I didn't really mind that. The initial objective was not to make an exact copy of argman's EAST in Keras but to create an implementation with similar performance.
